src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1933,6 +1933,7 @@ private void UndoOrganelleMoveAction(MicrobeEditorAction action)
         if (!isFreeToMove && MutationPoints < Constants.ORGANELLE_MOVE_COST)
         {
             CancelCurrentAction();
+            gui.OnInsufficientMp();
```

Comment:
I wonder if this could cause audio pops since `PlayInvalidActionSound` will also be called after this method is returning false. I might add a guard in `PlayCustomSound` and maybe `PlayButtonPressSound` too for good measure to disallow changing the audio stream while it's still playing (as mentioned in https://github.com/Revolutionary-Games/Thrive/issues/1268#issuecomment-801805657) in another PR just in case.

Reply:
I think maybe a better thing would be to split OnInsufficientMp into parts so that only the bar flashing part can be called here.

Reply:
For that probably a parameter could added to specify whether the sound should be played.